src/js/components/ContentInfo/contentInfo.test.js

```diff
@@ -18,7 +19,9 @@ describe('ContentInfo', () => {
   });
   it('should render correctly', () => {
     const wrapper = render(
-      <ContentInfo {...props} /> );
+      <MemoryRouter>
```

Comment:
What do you think about mocking `react-router`s components? Since we don't really need those rendered in snapshots.